<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contour</class>
 <widget class="QDialog" name="contour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>边缘查找阈值</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>轮廓检索模式</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>轮廓近似办法</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="maximum">
      <double>255.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>CV_RETR_EXTERNAL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CV_RETR_LIST</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CV_RETR_CCOMP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CV_RETR_TREE</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_2">
     <item>
      <property name="text">
       <string>CV_CHAIN_APPROX_NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CV_CHAIN_APPROX_SIMPLE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CV_CHAIN_APPROX_TC89_L1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CV_CHAIN_APPROX_TC89_KCOS</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
